include/Renderer.h

```diff
@@ -25,6 +27,12 @@ namespace ccm
 		 */
 		void draw(const Object& obj);
 
+		SDL_Texture* createTextureFromSurface(SDL_Surface* src);
+
+		void renderTexture(const Texture& source, const Rect dest);
```

Comment:
Rendering textures seems a bit odd to me. Would it make more sense to "apply" a texture to a renderable/object and just draw the object?

Reply:
Good point, I just did this to check what I had done was working, probably didn't need to be added to this PR, but will make the change to bind textures to objects